vumi/transports/smpp/processors/default.py

```diff
@@ -470,6 +470,10 @@ class SubmitShortMessageProcessorConfig(Config):
         "If `True`, messages longer than 140 bytes will be sent as a series "
         "of smaller messages with the user data headers. Default is `False`.",
         default=False, static=True)
+    multipart_sar_reference_rollover = ConfigInt(
+        "The maximum value to set for the reference number of a multi part "
+        "SMS. When this value is reached, the number will rollover to 0.",
+        default=0xFFFF, static=True)
```

Comment:
They're only the same for values of the form (2**n)-1. For example: `[i & 0xFE for i in range(260)]` -> `[0, 0, 2, 2, ..., 252, 252, 254, 254, 0, 0, 2, 2]`